src/spikeinterface/core/base.py

```diff
@@ -82,6 +82,10 @@ def get_num_segments(self) -> int:
         # This is implemented in BaseRecording or BaseSorting
         raise NotImplementedError
 
+    def get_parent(self) -> Optional[BaseExtractor]:
+        """Returns parent object if it exists, otherwise None"""
```

Comment:
@alejoe91 
I see, I vaguely remember that Sam does not like the word extractor either.

```suggestion
        """Returns the parent recording or sorter object if it exists, otherwise None.
        """
```

I have two suggestions to make this more specific less ambiguios: 
1) We can enchance the docstring here  as in my suggestion to disambiguate from parent class in the OOP sense. Maybe adding an example with recording and one with sorting.
2) Make this method private and then have a `parent_recording` and a `parent_sorting` that use `_get_parent` (maybe they could be attributes).